### DISCUSSION TITLE
how to generate a series of different images with a single prompt or how to create a batch

### DISCUSSION THREAD

**Discussion by oloboy** [mcmonkeyprojects/SwarmUI#337] [Q&A]
Hello everyone,

Could you help me understand how to generate a series of different images with a single prompt or how to create a batch?

For example, I would like to generate a realistic photo of a jumping cat, a running person, and a green tree — one image for each.

I tried using: realistic picture of <alternate: jumping cat, running person, green tree> and set the number of images to 3, but it generated 1 running person and 2 jumping cats, without the tree. Sometimes it only generates cats.

I also tried: realistic picture of <random: jumping cat, running person, green tree>, which worked better, but it selects randomly, without cycling through all the options consistently.

Where am I going wrong?

**Comment by kalebbroo** [comment]
What your looking for is <region

This feature is not finished and therefore not documented. The most instructions you will see in how to use it is the tooltip when you start typing.

**Comment by kwal559** [comment]
grid generator at bottom of screen may help.. or, you aren't limited to generating a picture and having to wait before generating another.  They stack up, you can add and remove presets, change prompt around, change your steps and CFG, it seems to keep track and keep on queuing..
